src/QuestionsForm.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { useForm } from 'react-hook-form'
+import questions from './questions.json'
+import styled from 'styled-components/macro'
+import TextInput from './TextInput'
+import NumberInput from './NumberInput'
+import CheckboxInput from './CheckboxInput'
+import RadioInput from './RadioInput'
+import Button from './Button'
+
+export default function QuestionsForm({ onClick }) {
+  const { register, handleSubmit } = useForm()
+
+  const onSubmit = (data) => {
+    console.log(data)
+    onClick(data)
+  }
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      {questions.map((question, index) => (
+        <React.Fragment key={index}>
```

Comment:
Dou you need to write "React.Fragment" or can you just use "<>" ?

Reply:
Yes, the "React.Fragment" is needed, because the "<>" doesn't support keys